biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.tsx

```diff
@@ -103,20 +105,21 @@ export const InputPage = () => {
           <b>throughout the town</b> so that our buildings can have an immediate
           energy source when they have a spike in energy consumption.
         </p>
-        <div className="startDatePicker">
-          <label>Start Date</label>
-          <DatePicker
-            showPopperArrow={false}
-            selected={startDate}
-            onChange={(date: Date) => setStartDate(date)}
-          />
-        </div>
-        <div className="endDatePicker">
-          <label>End Date</label>
-          <DatePicker
-            showPopperArrow={false}
-            selected={endDate}
-            onChange={(date: Date) => setEndDate(date)}
+        <div className="datePicker">
+          <label>Simulation Date Slider</label>
+          <ReactSlider
+            className="horizontal-slider"
+            onChange={(val) => {
+              const daysFromStart = (val as unknown) as number;
+              const newDate = new Date();
+              newDate.setDate(EARLIEST_DATE.getDate() + daysFromStart);
+              setStartDate(newDate);
+            }}
```

Comment:
can you change the end date or only the start date?

Reply:
Oh it is just simulating one day now

Reply:
If that's the case, why do we have `daysFromStart`? Should that be `hoursFromStart` instead?

Reply:
Oh thats days from last week, I.e. if it today is sunday, 1 means one day from last sunday, thus monday. Changed name to daysFromLastWeek and added comment